docs/getting-started/ios/fastlane-swift.md

```diff
@@ -0,0 +1,39 @@
+# Getting Started with Fastlane.swift
+
+Welcome to Fastlane.swift. Fastlane.swift allows you to write your _fastlane_ configuration using Xcode, in Swift - the language you have come to know and love from the world of iOS development..
```

Comment:
Why the `Welcome to Fastlane.swift`? Do we want the swift piece to be a natural thing that's just part of fastlane? It's just a language you choose during the onboarding process, right? If so, is this doc just here temporarily for the first people testing it?

Reply:
Yeah, this doc feels temporary for now, I think in the future we would work this in to just part of the normal getting started with fastlane docs where we would talk about selecting a language.

Reply:
Perfect, thanks for clarifying David, that makes lots of sense, let's make sure we don't forget to move this doc as part of the onboarding enhancements 👍

Reply:
👍  🚀

Reply:
Slack: `/remind me to move the docs in one month`?